### PR TITLE
added capability to view incentives that were redeemed for donations

### PR DESCRIPTION
**Pull request by jsstudz**

### CODE REVIEW

File: donation-ticker/index.html [hunk 1]
```diff
@@ -28,6 +28,7 @@ <h1 class="h5">{{ entity.sumDonations | formatMoney }} / {{ entity.fundraisingGo
 								<th scope="col">$</th>
 								<th scope="col">Name</th>
 								<th scope="col">Message</th>
+								<th scope="col">Incentive</th>
```
Comment (timmixell) [inline donation-ticker/index.html]: @jsstudz what do you think about dynamically including this via a `showIncentives` toggle in config.js?

**Review by antoniomistretta — APPROVED** [review]

File: config.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
 window.config = {
 
 	// the url of the DonorDrive API you wish to poll
-	api: 'https://docs.donordrive.com/api/',
+	api: 'https://www.extra-life.org/api/',
```
Comment (timmixell) [inline config.js]: this should still point to our "demo" site - while Extra Life constitutes a large percentage of our streamers, we have fundraisers for other orgs as well (and this participant wont go away at the end of the year)

File: donation-ticker.html [hunk 3]
```diff
@@ -27,6 +27,7 @@ <h1 class="h5">{{ entity.sumDonations | formatMoney }} / {{ entity.fundraisingGo
 								<th scope="col">$</th>
 								<th scope="col">Name</th>
 								<th scope="col">Message</th>
+								<th scope="col">Incentive</th>
```
Comment (timmixell) [inline donation-ticker.html]: i think i said it'd be "nice to have," but this column should either display based on configuration, OR if the fundraiser actually has incentives

**Review by timmixell — CHANGES_REQUESTED** [review]
@jsstudz, there appear to be EL-specific imagery and CSS in your commit. you can absolutely fork this repo and put those assets there, but the "official" version needs to be organization-agnostic.

additionally, it appears that you changed code that doesn't pertain to the original PR. the thermometer stuff belongs as a separate issue/pull request.

File: config.js (hunk 2 above)
Comment (jsstudz) [reply config.js]: You are absolutely right, accidentally left that in when testing based on my current fundraiser.

File: donation-ticker/index.html (hunk 1 above)
Comment (jsstudz) [reply donation-ticker/index.html]: That definitely makes sense.

**Pull request closed without merging**